rewriter/rewriter.cc

```diff
@@ -52,6 +53,7 @@ class Rewriterer {
         TypeMembers::run(ctx, classDef);
         Concern::run(ctx, classDef);
         TestCase::run(ctx, classDef);
+        RequiresAncestorKernel::run(ctx, classDef);
```

Comment:
Could you make this conditional on the feature being enabled?

Reply:
Ah, I hadn't considered making the running of the rewriter conditional, and was instead checking that inside the `run` method. I can move it out here for sure.

Reply:
Done!